tests/models/gptj/test_modeling_gptj.py

```diff
@@ -518,6 +527,48 @@ def test_model_from_pretrained(self):
             model = GPTJModel.from_pretrained(model_name, revision="float16", torch_dtype=torch.float16)
             self.assertIsNotNone(model)
 
+    @require_flash_attn
+    @require_torch_gpu
+    @pytest.mark.flash_attn_test
+    @slow
+    def test_flash_attn_2_generate_padding_right(self):
+        """
+        Overwritting the common test as the test is flaky on tiny models
+        """
+        model = GPTJForCausalLM.from_pretrained(
+            "EleutherAI/gpt-j-6b",
+            load_in_4bit=True,
+            device_map={"": 0},
+            revision="float16",
```

Comment:
The full version of GPT-J requires 48GB of RAM to load, so I used the float16 version provided by the model creators.

Reply:
Ok thanks!